Microsoft.Toolkit.Uwp.Input.GazeInteraction/GazeStats.cs

```diff
@@ -0,0 +1,105 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Windows.Foundation;
+
+namespace Microsoft.Toolkit.Uwp.Input.GazeInteraction
+{
+    /// <summary>
+    /// Provides basic stats for eye gaze
+    /// </summary>
+    public class GazeStats
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="GazeStats"/> class.
+        /// </summary>
+        /// <param name="maxHistoryLen">Defines the size of the circular history for the calculations of stats</param>
+        public GazeStats(int maxHistoryLen)
+        {
+            _maxHistoryLen = maxHistoryLen;
+            _history = new List<Point>();
+        }
+
+        /// <summary>
+        /// Resets the history, and stats
+        /// </summary>
+        public void Reset()
+        {
+            _sumX = 0;
+            _sumY = 0;
+            _sumSquaredX = 0;
+            _sumSquaredY = 0;
+            _history.Clear();
+        }
+
+        /// <summary>
+        /// Adds a new item to the history
+        /// </summary>
+        /// <param name="x">X axis of the new point</param>
+        /// <param name="y">Y axis of the new point</param>
+        public void Update(float x, float y)
+        {
+            var pt = new Point(x, y);
+            _history.Add(pt);
+
+            if (_history.Count > _maxHistoryLen)
+            {
+                var oldest = _history[0];
+                _history.RemoveAt(0);
+
+                _sumX -= oldest.X;
+                _sumY -= oldest.Y;
+                _sumSquaredX -= oldest.X * oldest.X;
+                _sumSquaredY -= oldest.Y * oldest.Y;
+            }
+
+            _sumX += x;
+            _sumY += y;
+            _sumSquaredX += x * x;
+            _sumSquaredY += y * y;
+        }
+
+        /// <summary>
+        /// Gets the mean of the history of points
+        /// </summary>
+        public Point Mean
+        {
+            get
+            {
+                var count = _history.Count;
```

Comment:
>_history.Count [](start = 28, length = 14)

Can this count be 0 ? Should we return a `Point(0, 0)` like done in `StandardDeviation`

Reply:
This class is not being used yet, and I'm keeping it here merely to be 100% compatible with what existed before in the C++ version. This class might be used in the future, so I will do the same logic as in StandardDeviation. I've also forgot to make this class internal, since it shouldn't be used publicly, at least not yet. Doing that now.